Author: aszlig <aszlig@nix.build>
Date:   Sat Apr 7 00:19:05 2018 +0200

    Remove Twitter reference from news
    
    Like always, let's get rid of anything that has to do with "social
    media".
    
    Signed-off-by: aszlig <aszlig@nix.build>
    Filename: no-twitter-in-news.patch

diff --git a/website/server/controllers/api-v3/news.js b/website/server/controllers/api-v3/news.js
index fe38dac2d..ce87b1cdc 100644
--- a/website/server/controllers/api-v3/news.js
+++ b/website/server/controllers/api-v3/news.js
@@ -3,7 +3,7 @@ import { authWithHeaders } from '../../middlewares/auth';
 let api = {};
 
 // @TODO export this const, cannot export it from here because only routes are exported from controllers
-const LAST_ANNOUNCEMENT_TITLE = 'SHIMMER HAIR, PASTEL SKINS, HABITICA FORGE TWITTER, AND GUILD SPOTLIGHT';
+const LAST_ANNOUNCEMENT_TITLE = 'SHIMMER HAIR, PASTEL SKINS AND GUILD SPOTLIGHT';
 const worldDmg = { // @TODO
   bailey: false,
 };
@@ -38,9 +38,6 @@ api.getNews = {
         <h3>Shimmer Hair Colors and Pastel Skin Set</h3>
         <p>The Seasonal Edition Shimmer Hair Colors and Pastel Skin Set are now available for purchase in User > Edit Avatar! These skin sets will only be available to purchase until April 30th, and then they will disappear from the shop until next Spring Fling. If you buy them, though, you will have access to them year-round!</p>
         <div class="small mb-3">by Lemoness and McCoyly</div>
-        <h3>The Habitica Forge</h3>
-        <p>We've launched an exciting new Twitter account! <a href='https://twitter.com/habiticaforge' target='_blank'>Habitica Forge</a> will be tweeting automated updates about all the work of our fantastic blacksmiths. If you're curious about what fixes and improvements are in the works, be sure to check it out! This account will also be the official source for updates from Habitica about any site or app access issues or outages.</p>
-        <div class="small mb-3">by Beffymaroo, TheHollidayInn, Paglias, Alys, Viirus, SabreCat, Blade, and all Habitica's wonderful Blacksmiths!</div>
         <h3>Be the Change: Guilds for Making a Difference</h3>
         <p>There's a new <a href='https://habitica.wordpress.com/2018/04/05/be-the-change-guilds-for-making-a-difference/' target='_blank'>Guild Spotlight on the blog</a> that highlights the Guilds that can help you as you work to make the world a better place! Check it out now to find Habitica's best communities for volunteers and general do-gooders.</p>
         <div class="small mb-3">by Beffymaroo</div>
